t/03.scripts/beekeeper_reset.t

```diff
@@ -96,7 +96,7 @@ foreach my $pipeline_url (@$ehive_test_pipeline_urls) {
     my $completion_date1 = fetch_job_completion_date($job_adaptor, $job_id);
 
     # Try again
-    ok(system($ENV{'EHIVE_ROOT_DIR'}.'/scripts/runWorker.pl', '--job_id', $job_id), 'Cannot rerun a job that is already done without the --force option');
+    runWorker($pipeline_url, ['--job_id' => $job_id, '--force']);
```

Comment:
Just to make sure my understanding is correct - what I think is going on is this: this line and the following assert_jobs weren't really doing much, beyond verifying that runWorker would respect --job_id. (it was trying to get runWorker to complain that it couldn't re-run a done job without --force, but there was nothing to observe because runWorker exits with 0.) Job 4 would still be done, so assert_jobs would show that no other jobs changed status. With your change, you're showing that you can try and force the re-run of a done job, so the 0 exit code now makes sense, and then go on to verify that no jobs changed status. Is that correct?

Reply:
Wait a sec. I think I've missed that just below there is another `runWorker` call on the same job_id with `--force` too. I think this line shouldn't have `--force`, and we check that the completion date remains the same. Then the next call will use `--force` and we can check that the completion date has changed